package.json

```diff
@@ -74,7 +74,7 @@
     "run-chromatics": "node -r esm ./scripts/run-chromatics.js",
     "serve-storybooks": "http-server ./built-storybooks -p 8001",
     "start": "yarn workspace official-storybook storybook --no-manager-cache",
-    "test": "NODE_OPTIONS=--max_old_space_size=4096 jest",
+    "test": "NODE_OPTIONS=\"--max_old_space_size=4096 --experimental-vm-modules\" jest",
```

Comment:
wait WHUT?

Is this needed?

Reply:
Now @mdx/react is ESM only. See [Migrating from v1 to v2](https://mdxjs.com/migrating/v2/#esm).

Using `--experimental-vm-modules` is one of the options given by jest [docs](https://jestjs.io/docs/ecmascript-modules).

Reply:
Ow I see... 👍